woocommerce/woocommerce-ios
# Custom help content for "Not a WooCommerce site" error screen

Part of: #7547 

### Description
As a part of improving the help center content, this PR adds custom content with FAQs and Answers for the "Not a WooCommerce site" error at the following URL.

https://woocommerce.com/document/android-ios-apps-login-help-faq/#not-a-woocommerce-site

Internal ref - pe5sF9-sZ-p2

### Testing instructions

1. Create a JN site with the Jetpack plugin and don't install the WooCommerce plugin.
1. Install and launch the app.
1. Tap `Skip` to reach the login prologue screen.
1. Tap the `Enter Your Store Address` button and enter the site address.
1. Tap `Continue` and finish the sign-in process. (Either using the .org site or WPCOM credentials)
1. You will land in the following "Not a WooCommerce site" error screen. 

<img src="https://user-images.githubusercontent.com/524475/192209941-86f6f028-3b20-4170-a5ad-210575f4122b.png" width="300"/>

7. Tap `Help` -> `Help Center`
1. Observe that you are directed to https://woocommerce.com/document/android-ios-apps-login-help-faq/#not-a-woocommerce-site in a web view
1. In Xcode debug console observe that the following event is tracked. Note that the `source_flow` value will differ based on the flow you selected to login.
```
Tracked support_help_center_viewed, properties: [AnyHashable("source_step"): "not_woo_store", AnyHashable("help_content_url"): "https://woocommerce.com/document/android-ios-apps-login-help-faq/#not-a-woocommerce-site", AnyHashable("source_flow"): "login_password_magic_link_emphasis"]
```

---
